pom.xml

```diff
@@ -25,6 +25,8 @@
         <module>6-0-test-driven-development</module>
         <module>java-fundamentals-util</module>
         <module>lesson-demo</module>
+        <module>7-0-java-concurrency</module>
```

Comment:
@tkuzub 
* please move this line up (right after the 6th module)
* please remove the next line (you should follow the same approach we have), in other words, **this pom file declares only high-level modules**. And `7-0-0-hello-threads` should be declared as a submodule of `7-0-java-concurrency` inside its pom file.